File: temp/fib.py
import time

def fibonacci(n):
    if n <= 1:
        return n
    return fibonacci(n - 1) + fibonacci(n - 2)

n = 45
inicio = time.time()
resultado = fibonacci(n)
fim = time.time()

print(f"Fibonacci({n}) = {resultado}")
print(f"Tempo de execucao: {fim - inicio:.6f} segundos")

File: temp/fib.c
#include <stdio.h>
#include <time.h>

long long fibonacci(int n) {
    if (n <= 1) return n;
    return fibonacci(n - 1) + fibonacci(n - 2);
}

int main() {
    int n = 45;
    clock_t inicio = clock();
    long long resultado = fibonacci(n);
    clock_t fim = clock();

    double tempo_exec = (double)(fim - inicio) / CLOCKS_PER_SEC;

    printf("Fibonacci(%d) = %lld\n", n, resultado);
    printf("Tempo de execucao: %.6f segundos\n", tempo_exec);
    return 0;
}